### ISSUE TITLE
wrong message about engines field

### ISSUE THREAD

**Issue by XhmikosR** [ljharb/ls-engines#4]
```
C:\Users\xmr\Desktop\hugo-bin>ls-engines
`node_modules` found; loading tree from disk...

Valid node version range: ^13 || ^12 || ^11 || ^10 || ^9 || ^8 || ^7 || ^6

Currently available latest releases of each valid node major version: v13.7.0, v12.14.1, v11.15.0, v10.18.1, v9.11.2, v8.17.0, v7.10.1, v6.17.1

Current node version, v12.14.1, is valid!

Your “engines” field is either missing, or set to ”*”! Prefer explicitly setting a supported engine range.

You can fix this by running `ls-engines --save`, or by manually adding the following to your `package.json`:
"engines": {
  "node": "^13 || ^12 || ^11 || ^10 || ^9 || ^8 || ^7 || ^6"
}
```

The package.json file does have an engines property though:

https://github.com/fenneclab/hugo-bin/blob/a8c6fb39bdb2bcb559367c9f7e71cac60404f7f9/package.json#L36

Did I miss something?

**Comment by ljharb** [comment]
hmm, seems like a bug alright :-)

**Comment by ljharb** [comment]
Released in v0.3.1.
